modules/nf-core/iqtree/tests/iqtree_sup.config

```diff
@@ -0,0 +1,5 @@
+process {
+    withName: "IQTREE" {
+        ext.args = "-t https://raw.githubusercontent.com/nf-core/test-datasets/phyloplace/testdata/cyanos_16s.newick -sup https://raw.githubusercontent.com/nf-core/test-datasets/phyloplace/testdata/cyanos_16s.newick"
```

Comment:
Files generally should never be supplied through ext.args as it by passes Nextflow's staging mechanism and can lead to multiple access issues

Reply:
Good catch, thank you! This is a leftover from testing which I missed. This config file actually shouldn't be necessary anymore, I'll just remove it.